src/tcp-intercept.cxx

```diff
@@ -22,6 +22,8 @@ std::string logfilename;
 FILE *logfile;
 
 static const int MAX_CONN_BACKLOG = 32;
+bool KEEPALIVE = false;
+bool NODELAY = false;
```

Comment:
I prefer lowercase variables (vs uppercase constants), but this is a minor issue.
Also: group them with the 2 vars below (line 28,29) instead of with the constants above (line 24)